a11y-dialog.js

```diff
@@ -26,17 +29,24 @@
    * @param {Element} node
    * @param {(NodeList | Element | string)} targets
    */
-  function A11yDialog(node, targets) {
+  function A11yDialog(node, targets, params) {
```

Comment:
This probably introduces the new API for v6, where everything will be on a single options object.